app/components/address-mapper.tsx

```diff
@@ -7,6 +7,7 @@ import Heading, { HeadingProps } from "./heading";
 import Map from "./map";
 import Report from "./report";
 import Information from "./information";
+import { FeatureCollection, Geometry } from "geojson";
 
 const addressLookupCoordinates = {
```

Comment:
we can probably simplify this later since the structure was added mainly for reference